### PR TITLE
Link RS doc in Deployment doc

### PR DESCRIPTION
**Pull request by janetkuo**
Published at http://janetkuo.github.io/docs/user-guide/deployments/

@kubernetes/docs

### CODE REVIEW

**Comment by janetkuo** [comment]
cc @madhusudancs

File: docs/user-guide/deployments.md [hunk 1]
```diff
@@ -6,24 +6,24 @@
 
 ## What is a _Deployment_?
 
-A _Deployment_ provides declarative updates for Pods and ReplicaSets.
-You only need to describe the desired state in a Deployment object, and the deployment
+A _Deployment_ provides declarative updates for [Pods](/docs/user-guide/pods/) and [Replica Sets](/docs/user-guide/replicasets/)(the next-generation Replication Controller).
```
Comment (pwittrock) [inline docs/user-guide/deployments.md]: nit: space after replica sets

**Comment by johndmulhausen** [comment]
Agreed, you gotta add that space. Then, we're ready to go here.

**Comment by janetkuo** [comment]
Fixed.

**Pull request merged**